SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -365,3 +365,18 @@ CONNECTIONMANAGER_TEST(SetupFromCommandLine_Receptionist_URLAndExternalBridge)
 
 	return true;
 }
+
+CONNECTIONMANAGER_TEST(SetupFromCommandLine_Receptionist_SetupConnectionConfigFromURL)
+{
+	// GIVEN
+	FTemporaryCommandLine TemporaryCommandLine("");
+	USpatialConnectionManager* Manager = NewObject<USpatialConnectionManager>();
+
+	// WHEN
+	const bool bSuccess = Manager->TrySetupConnectionConfigFromCommandLine("");
```

Comment:
Hmm, what's the expected IP that we connect with here? Localhost? In which case I would include it as an expectation in the test. (Also assuming this will use the receptionist by default).

Reply:
Done!

Reply:
I don't think we need this Miron. If we fail to "TrySetupConnectionConfigFromCommandLine" who cares what the host is? In the actual connection flow, if we fail TrySetupConnectionConfigFromCommandLine we will attempt to connect using the URL.

Reply:
Ah, you are correct, I missed the `bSuccess == false`.
Okay, we can remove the localhost again (sorry about that!).
If we do want to go really pedantic, we could test the new case, no host but specified port, which I guess should be successful and localhost?

Reply:
Nevermind, already exists here `CONNECTIONMANAGER_TEST(SetupFromCommandLine_Receptionist_ReceptionistHostLocal)`